i18n/locale-info.php

```diff
@@ -3,326 +3,3980 @@
  * Locales information
  *
  * @package WooCommerce\i18n
- * @version 3.5.0
+ * @version 5.5.0
```

Comment:
Can you please change the version to 5.6.0

Reply:
Done.